styles.css

```diff
@@ -0,0 +1,920 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #344563;
+  --clr-gray-100: #b3bac5;
+}
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo a {
+  color: var(--clr-purple);
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  background-color: var(--clr-white);
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  align-items: center;
+}
+
+.dev a {
+  font-weight: 500;
+  color: #7f8cff;
+}
+
+.nav-link a {
+  color: #fff;
+}
+
+.social-icons a {
+  margin-right: 1.2rem;
+}
+
+.nav-link a:hover {
+  color: var(--clr-para);
+}
+
+.menu-icon > span {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: var(--clr-purple);
+}
```

Comment:
[OPTIONAL] Kindly add some blur to your menu so it's similar to the Figma design. This will help your portfolio have a more unique style and demonstrate your attention to detail 👍 . You can use something like `backdrop-filter: blur(5px)`. More info here: https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter

Figma design:
![xtempl](https://user-images.githubusercontent.com/111214189/225990588-e66902cc-a404-4bcb-99bf-b2739d81df5e.png)


Having your menu almost identical to your chosen Figma template is one of the requirements, but the previous reviewer didn't notice it so I requested this as optional but I highly recommend you make the changes.